docker/development-easy-arm/docker-compose.yml

```diff
@@ -12,6 +12,18 @@ services:
     - databasevolume:/var/lib/mysql
     environment:
       MYSQL_ROOT_PASSWORD: root
+    healthcheck:
+      test:
+      - CMD
+      - /usr/local/bin/healthcheck.sh
```

Comment:
For other's benefits the healthcheck.sh exists on the MySQL docker, will check the system using the MySQL user and verify that connection and innodb is initialized. More info here: https://mariadb.com/kb/en/using-healthcheck-sh/